pkg/storaged/content-views.jsx

```diff
@@ -326,7 +345,63 @@
         });
     }
 
+    function repair() {
+        const vgroup = lvol && client.vgroups[lvol.VolumeGroup];
+        if (!vgroup)
+            return;
+
+        const summary = client.lvols_stripe_summary[lvol.path];
+        const missing = summary.reduce((sum, sub) => sum + (sub["/"] ?? 0), 0);
+
+        function usable(pvol) {
+            // must have some free space and not already used for a
+            // subvolume other than those that need to be repaired.
+            return pvol.FreeSize > 0 && !summary.some(sub => !sub["/"] && sub[pvol.path]);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230913-131211-e9281c3e-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)